source/physics/Electroluminescence.h

```diff
@@ -10,7 +10,7 @@
 #define ELECTROLUMINESCENCE_H
 
 #include <G4VDiscreteProcess.hh>
-
+#include <G4PhysicsOrderedFreeVector.hh>
```

Comment:
Since you had to add the `include` here, you can delete it from the source file. The same for the other classes.